### ISSUE TITLE
Handling alerts

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#180]
It looks like the alert element has area-hidden is toggled in SparkNode. Besides abstraction (should be handled in https://github.com/phetsims/john-travoltage/issues/171), I'm curious at how this is set up on a step listener, and whether SparkNode has sole control of the alert element (it behaves like it does).

If multiple elements "controlled" the alert element in step listeners, it seems like the "alert" from one could then immediately afterwards be "hidden" by another element.

Is there documentation on why hidden is toggled, and how alert/status indicators work?

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
> I'm curious at how this is set up on a step listener, and whether SparkNode has sole control of the alert element (it behaves like it does).

It is no longer hooked up to the step listener.  An alert is triggered when the discharge starts and ends.  SparkNode is the only thing that controls the alert element in this sim. That may not be the case for other sims.

> Is there documentation on why hidden is toggled, and how alert/status indicators work?

Hidden is no longer toggled, nor supported by AriaHerald.  All alerts can still be disabled if necessary.  Documentation for how alert/ status indicators work is in [AriaHerald](https://github.com/phetsims/scenery-phet/blob/master/js/accessibility/AriaHerald.js).

Assigning to @jonathanolson for further discussion if necessary.

**Comment by jessegreenberg** [comment]
This issue is now stale and utteranceQueue is used to manage all alerts.
